### ISSUE TITLE
3.13 Unsupported

### ISSUE THREAD

**Issue by HEROgold** [Klavionik/copykitten#8]
Just tried to install for 3.13, any reason why this isn't supported? Or is it simply because it has not been tested for that version yet?

**Comment by Klavionik** [comment]
Hey! Sorry for the inconvenience. I've updated the package to allow installation on Python 3.13 and higher. I can't run the test suite using 3.13 yet, but I suppose it should work fine.

**Comment by HEROgold** [comment]
I just installed succesfully.
Also ran a pytest, had some warning about Unknown pytest.mark.repeat. Couldn't test any futher.

**Comment by Klavionik** [comment]
Yes, running the tests requires a few packages. For example Pillow, which doesn't support 3.13 yet. That's why I have to wait for a while before I can make the test suite run on 3.13.

However, as the actual clipboard operations are provided by a Rust library, using `copykitten` with 3.13 shouldn't be different from 3.12.
